e2e/e2e.go

```diff
@@ -223,3 +223,17 @@ func WaitForTransaction(hash common.Hash, client *ethclient.Client, timeout time
 		}
 	}
 }
+
+func UpdateSlackUrls(cfg *alert.Config, url string) {
+	for i := range cfg.AlertRoutingParams.AlertRoutes.High.Slack {
+		cfg.AlertRoutingParams.AlertRoutes.High.Slack[i].URL = url
+	}
+
+	for i := range cfg.AlertRoutingParams.AlertRoutes.Medium.Slack {
+		cfg.AlertRoutingParams.AlertRoutes.Medium.Slack[i].URL = url
+	}
+
+	for i := range cfg.AlertRoutingParams.AlertRoutes.Low.Slack {
+		cfg.AlertRoutingParams.AlertRoutes.Low.Slack[i].URL = url
+	}
```

Comment:
Why is this function necessary?

Reply:
The testhttp server uses a different port every time, so this just updates the params to use the correct server url. Could also find a way to set a static port on instantiation of the server however